logisland-core/logisland-framework/logisland-utils/src/main/java/com/hurence/logisland/util/runner/StandardProcessorTestRunner.java

```diff
@@ -82,7 +82,11 @@ public ProcessContext getProcessContext() {
 
     @Override
     public void run() {
-        this.processor.init(context);
+        try {
+            this.processor.init(context);
```

Comment:
this method *run* should not catch this error and just throwing it in my opinion